src/Sylius/Bundle/CoreBundle/EntityObserver/CreateLogEntryOnPriceChangeObserver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\EntityObserver;
```

Comment:
```suggestion
namespace Sylius\Bundle\CoreBundle\PriceHistory\EntityObserver;
```
Similar to CatalogPromotion, group services associated with price history into `PriceHistory` directory